service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocmosisTemplateDataGeneration.java

```diff
@@ -4,38 +4,29 @@
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 
 import java.io.IOException;
-import java.io.InputStream;
-import java.util.Base64;
 import java.util.Map;
 
-import static java.lang.String.format;
+import static uk.gov.hmcts.reform.fpl.enums.DocmosisImages.COURT_SEAL;
+import static uk.gov.hmcts.reform.fpl.enums.DocmosisImages.CREST;
+import static uk.gov.hmcts.reform.fpl.enums.DocmosisImages.DRAFT_WATERMARK;
 
 public abstract class DocmosisTemplateDataGeneration {
-    static final String BASE_64 = "image:base64:%1$s";
 
-    // REFACTOR: 05/12/2019 make not static when all document generation uses this abstract class
-    public static String generateDraftWatermarkEncodedString() throws IOException {
-        InputStream is = DocmosisTemplateDataGeneration.class.getResourceAsStream("/assets/images/draft-watermark.png");
-        byte[] fileContent = is.readAllBytes();
-        return Base64.getEncoder().encodeToString(fileContent);
-    }
-
-    public static String generateCourtSealEncodedString() throws IOException {
-        InputStream is = DocmosisTemplateDataGeneration.class
-            .getResourceAsStream("/assets/images/family-court-seal.png");
-        byte[] fileContent = is.readAllBytes();
-        return Base64.getEncoder().encodeToString(fileContent);
+    protected static Map<String, Object> getDraftWaterMarkData() {
```

Comment:
Do you want to add the options for the large and small hmcts logos here? Might be useful to have all the methods in one place and consistent

Reply:
It seems that all the methods in the class are used by the subclasses. DocmosisCoverDocumentsService where the logos are used is not one of the subclasses. So all the methods could be moved to separate class/interface, where all the images are retrieve from.  Then DocmosisTemplateDataGeneration could be turned into and interface. 
Or DocmosisCoverDocumentsService can be refactored to use DocmosisTemplateDataGeneration base class.

Reply:
Let me know your thoughts

Reply:
I think eventually it would be good for all of our code that generates template data for docmosis documents are in classes that extend DocmosisTemplateDataGeneration personally, that is initially what I had in mind when I first made it

Reply:
Mainly because most a lot of the code that was for generating the data ended up taking up large parts of their respective service classes and as such there was just too much going on

Reply:
So maybe tech debt for a later date

Reply:
Ok I will add the methods then put a todo comment for the mentioned refactor.